<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreeDView</class>
 <widget class="QDialog" name="ThreeDView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D view</string>
  </property>
  <widget class="glWidget" name="myGlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="xRotSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="yRotSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>760</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSlider" name="zRotSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>790</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>xRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGlWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGlWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>769</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>652</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGlWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>808</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>682</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
